cpp/CMakeLists.txt

```diff
@@ -2,8 +2,9 @@ cmake_minimum_required(VERSION 3.19)
 set(CMAKE_POLICY_DEFAULT_CMP0077 NEW) # option respects set variables
 project(tosca)
 
-option(TOSCA_ASAN "Enable AddressSanitizer for Tosca targets.")
-option(TOSCA_ASSERT "Enable Tosca specific assertions.")
+option(TOSCA_ASAN "Enable AddressSanitizer for Tosca targets." OFF)
```

Comment:
why is default off ?

Reply:
I don't want to instrument code by default. 
The default build shall be not-instrumented  (either sanitizer nor coverage) 
(this defaults only take effect if you call CMake directly, the top level Makefile will always pre-fill them.